lang/es.json

```diff
@@ -200,17 +200,17 @@
                 "slashRoleNewRoleOption": "Nuevo rol DJ"
             },
             "filter": {
-                "availableFilters": "Available Filters",
-                "currentlyUsedFilters": "Currently Used Filters",
-                "currentState": "`{filter}` is currently `{state}`",
-                "description": "Configure song filter",
-                "embedFooter": "To {opstate, select, enable {enable} other {disable}} the filter, use {prefix}filter {opstate} {filter}",
-                "filterSet": "`{filter}` filter is now `{state}`",
-                "slashStateDescription": "{state, select, enable {Enable} other {Disable}} a song filter",
-                "slashStateFilterDescription": "The song filter that will be {state, select, enable {enabled} other {disabled}}",
-                "slashStatusDescription": "Check the current status of song filters",
-                "slashStatusFilterDescription": "The song filter that you want to check",
-                "specifyFilter": "Please specify the song filter"
+                "availableFilters": "Filtros disponibles",
+                "currentlyUsedFilters": "Filtros utilizados actualmente",
+                "currentState": "`{filter}` es actualmente `{state}`",
+                "description": "Configurar el filtro de canciones",
+                "embedFooter": "Para {Optar, seleccionar, activar {enable} o {disable}} el filtro, utilice {prefix}filter {opstate} {filter}",
+                "filterSet": "El filtro `{filter}` ya está `{state}`",
+                "slashStateDescription": "{establecer, seleccionar, activar {Enable} o {Disable}} un filtro de canciones",
```

Comment:
The `{state, select, enable {...} other {...}}` is messageformat syntax for switch-case (or if-else).

So, take `{run, select, yes {He's running} other {He's not running}}` for example.
It will check the content of `run` variable. If the content is `yes` then it'll write `He's running`. But if not, it will write `He's not running`.

In the case of ours (`{state, select, enable {Enable} other {Disable}}`), I have set that `state` variable will only have two possible values/contents. Those are `enable` and `disable`. Put the translated string in the inner bracket according to the state it is in.

For example, we want to write `Foo` if state is `enable` and `Bar` is state is `disable`. We will have this in the string: `{state, select, enable {Foo} other {Bar}}`.

Reply:
For more information, you can look at messageformat docs about SelectFormat here
https://messageformat.github.io/messageformat/v2/page-guide#toc3

Reply:
Okay thanks, I'll look into it.